agoradatatools/etl/transform.py

```diff
@@ -137,6 +151,105 @@ def transform_network(datasets: dict):
+    gene_metadata['igap'] = gene_metadata.apply(lambda row: False if row['hgnc_symbol'] is NaN else True, axis=1)
+    gene_metadata['igap'].fillna(False, inplace=True)
+    gene_metadata = gene_metadata[['ensembl_gene_id', 'symbol', 'name', 'summary', 'alias', 'igap']]
+
+    gene_metadata = pd.merge(left=gene_metadata, right=eqtl, how='left', on='ensembl_gene_id')
+    gene_metadata = gene_metadata[['ensembl_gene_id', 'symbol', 'name', 'summary', 'alias', 'igap', 'haseqtl']]
+    gene_metadata.rename(columns={'haseqtl': 'eqtl'}, inplace=True)
+    gene_metadata['eqtl'] = gene_metadata['eqtl'].replace({'TRUE': True}).fillna(False)
+
+    rna_change = rna_change[['ensembl_gene_id', 'adj_p_val']]
+    rna_change = rna_change.groupby('ensembl_gene_id')['adj_p_val'].agg('min').reset_index()
+
+    gene_metadata = pd.merge(left=gene_metadata, right=rna_change, how='left', on='ensembl_gene_id')
+    gene_metadata['adj_p_val'] = gene_metadata['adj_p_val'].fillna(-1)
+    gene_metadata['rna_brain_change_studied'] = gene_metadata.apply(
+        lambda row: False if row['adj_p_val'] == -1 else True, axis=1)
+    gene_metadata['rna_in_ad_brain_change'] = gene_metadata.apply(
+        lambda row: True if row['adj_p_val'] <= adjusted_p_value_threshold else False, axis=1)
+
+    gene_metadata = gene_metadata[
+        ['ensembl_gene_id', 'name', 'summary', 'alias', 'igap', 'symbol', 'eqtl', 'rna_in_ad_brain_change',
+         'rna_brain_change_studied']]
+
+
+    proteomics = proteomics.groupby('ensg')['cor_pval'].agg('min').reset_index()
+
+    gene_metadata = pd.merge(left=gene_metadata, right=proteomics, how='left', left_on='ensembl_gene_id',
+                             right_on='ensg')
+    gene_metadata['cor_pval'] = gene_metadata['cor_pval'].fillna(-1)
+    gene_metadata['protein_brain_change_studied'] = gene_metadata.apply(
+        lambda row: False if row['cor_pval'] == -1 else True, axis=1)
+    gene_metadata['protein_in_ad_brain_change'] = gene_metadata.apply(
+        lambda row: True if row['cor_pval'] <= protein_level_threshold else False, axis=1)
+
+    gene_metadata = gene_metadata[
+        ['ensembl_gene_id', 'name', 'summary', 'symbol', 'alias', 'igap', 'eqtl', 'rna_in_ad_brain_change',
+         'rna_brain_change_studied', 'protein_in_ad_brain_change', 'protein_brain_change_studied']]
+    gene_metadata.rename(columns={'symbol': 'hgnc_symbol'}, inplace=True)
+
+    return gene_metadata
+
+
+def transform_gene_info(datasets: dict):
+
+    gene_metadata = datasets['syn26868788']
+    target_list = datasets['syn12540368']
+    median_expression = datasets['syn12514804']
+    drugability = datasets['syn13363443']
+
+    # these are the interesting columns of the drugability dataset
+    useful_columns = ['geneid', 'sm_druggability_bucket', 'safety_bucket', 'abability_bucket', 'pharos_class',
+                      'classification', 'safety_bucket_definition', 'abability_bucket_definition']
+    drugability = drugability[useful_columns]
+
```

Comment:
This should be 'druggability' not 'drugability'...is it better to change that here or via the column_rename functionality, given that we didn't derive this key from source data?

Reply:
It's better to change it here!  Good catch!